lib/foundation_rails_helper/form_builder.rb

```diff
@@ -129,6 +129,42 @@ def custom_label(attribute, text, options, &block)
       label(attribute, text, options)
     end
 
+    def tag_from_options(name, options)
+      if options && options[:size].present? && options[:value].present?
+        content_tag(:div,
+                    content_tag(:span, options[:value], :class=>name),
+                    :class=>"small-#{options[:size]} large-#{options[:size]} columns")
+      else
+        "".html_safe
+      end
+    end
+
+    def calculate_label_size(prefix_options, postfix_options)
```

Comment:
This name is confusing.  It is the input's column size, not the label's column size.